registration-processor/pre-processor/registration-processor-cmd-validator-stage/src/test/java/io/mosip/registration/processor/cmdvalidator/DeviceValidatorTests.java

```diff
@@ -217,4 +221,17 @@ public void deviceValidationNoBiometricsInPacket()
 				any(), any(),any())).thenReturn(null);
 		deviceValidator.validate(regOsi, process, registrationId);
 	}
+	
+	@Test
+	public void deviceValidationNoOfficerBiometricsInPacket() 
+			throws JsonProcessingException, ApisResourceAccessException, IOException, 
+				BaseCheckedException, JSONException {
+		Map<String, String> metamap = new HashMap<>();
+		JSONArray jsonArray = new JSONArray();
+		JSONObject jsonObject1 = new JSONObject();
+		jsonObject1.put("officerBiometricFileName", "null");
```

Comment:
Why this is not actual null ?

Reply:
there is some restriction in jsonObject to use null value... thats why i used "null". but now i got the solution we can use JSONObject.NULL.